### ISSUE TITLE
exemplar set

### ISSUE THREAD

**Issue by pp1016** [xialeiliu/Long-Tailed-CIL#11]
Hi, I have some questions regarding the exemplar set. 
(1) Did you use fixed memory (M=2000) or growing memory (20 per class) for experiments? From your paper, it looks like you are using growing memory but in the Section3.1, your description is about fixed memory with total exemplar size |M|. 
(2) For the conventional setup, is the training epochs, learning rate decay ,...other training parameters the same as in LT or LTIO? Still the growing memory? 
(3) Is there any reason to use growing memory? Since fixed memory is more popular in CIL. 

Thanks and look forward to your response!

**Comment by Ethanhuhuhu** [comment]
Thanks for your interest.
For (1), we use growing memory, as our experiments are on 50 based scenarios. The discussion in section 3.1 is an introduction example.
For (2), all the learning rate is same.
For (3), growing memory is always used in 50 based scenarios while fixed memory in 0 based scenarios.
